sBTC SIP

```diff
@@ -0,0 +1,166 @@
+
+## Introduction
+
+## Glossary
+
+| Term                     | Definition |
+|--------------------------|------------|
+| SIP-10 Token             | A token on the Stacks blockchain that adheres to the fungible token standards outlined in SIP-10. |
+| sBTC                     | A SIP-10 token on the Stacks Blockchain that can be turned back into BTC on the Bitcoin Blockchain. 1 sBTC is equivalent to 1 BTC on the Bitcoin Blockchain. |
+| sBTC operation           | An operation that initiates some action from the sBTC protocol. |
+| Bitcoin holder           | A person or system holding BTC, interested in acquiring sBTC. |
+| sBTC holder              | A person or system holding sBTC, interested in turning it back into BTC. |
+| .sbtc contract           | A smart contract (or a collection of contracts) defining the sBTC token and functions related to it. |
+| sBTC UTxO                | The single UTxO holding the entire BTC balance that’s pegged into sBTC. This UTxO is managed and maintained by the sBTC Signers. |
+| Stacks Signer            | The standard Stacks Signer introduced in Nakamoto. |
+| sBTC Bootstrap Signer    | A completely separate signer from the Stacks signer that will sign sBTC operations and communicate with contracts on the chain to make that feasible. This entity is registered as an sBTC signer and has partial access to spending the sBTC UTxO. This only exists in sBTC-v1. In subsequent sBTC releases this functionality will be native in the Stacks Signer. |
+| sBTC Signer              | sBTC Signer is a more general term for the entity that signs sBTC transactions under any version of sBTC. Under sBTC-v1 the sBTC signer is the sBTC Bootstrap Signer which is wholly separate from the Stacks Signer, but in subsequent releases of sBTC the Stacks Signer will also be responsible for signing sBTC operations, so it would be the sBTC Signer. |
+| sBTC Bootstrap Signer Set| The set of all sBTC signers. Each is registered with the .sbtc contract and the transfer. These entities as a group have full democratic access to the sBTC UTxO. |
+| sBTC Signer API          | API exposed by the sBTC Signer binary that handles basic low level commands. Remains private except to the Deposit API. |
+| Deposit API              | A third party API that communicates with the sBTC Bootstrap Signers via the sBTC Signer API. |
+
+## Problem Statement
+Bitcoin is limited in its programmability and scalability. While its security and censorship-resistance make it an ideal platform to build decentralized applications, it is not able to scale to meet the modern needs of users and developers. These challenges have been largely driven by the ethos of the Bitcoin ecosystem to prioritize simplicity and decentralization at the base layer.
```

Comment:
Again, I'd drop scalability as a goal (since it's not going to be achieved).

Reply:
Scalability is an explicit goal of the system and I'd argue it belongs in the SIP.

Reply:
Scalability is not achievable in blockchains, so I must insist that this be dropped.  I will not sign off on a SIP that promises something impossible.